packages/wallets/src/features/cashier/modules/WithdrawalCrypto/WithdrawalCrypto.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { WalletText } from '../../../../components';
+import { TransactionStatus } from '../TransactionStatus';
+import { WithdrawalCryptoForm, WithdrawalDisclaimer } from './components';
+import './WithdrawalCrypto.scss';
+
+type TWithdrawalCryptoProps = {
+    verificationCode?: string;
+};
+
+const WithdrawalCrypto: React.FC<TWithdrawalCryptoProps> = ({ verificationCode }) => {
+    return (
+        <div className='wallets-withdrawal-crypto-container'>
```

Comment:
typically, on top level of component, it should have "wallets-withdrawal-crypto", 
whats the reason for wrapping it this way, not the other way around?

Reply:
by "other way around" I mean putting -container iside "walets-withdrawal-crypto"